recipes/stages/classification/semisl.yaml

```diff
@@ -20,3 +21,6 @@ custom_hooks:
   priority: ABOVE_NORMAL
 - type: SemiSLClsHook
 
+task_adapt:
```

Comment:
This seems like something worth discussing.

Reply:
I agree, please share your nice thinks @jaegukhyun , @supersoob , @goodsong81

Reply:
I don't get what is the point of discussing? If it is about adding task_adapt flag to semisl recipe, we agreed on semi-sl should support task_adapt and ignore label.

Reply:
As far as I know it is most likely not supported in semi right away. In particular, ignore labels are not supported due to differences in loss.

Reply:
Yes, I think we can resolve this conversation as-is.

Reply:
Okay, then @JihwanEom do you have plan to solve task adapt issue and ignore label issue in semi-sl cls in this PR or in this sprint?

Reply:
No, I'm not mainly responsible, only focusing on recovering semi-sl in otx. @sovrasov might be responsible for future works in multi-class semi sl.